marshmallow_jsonapi/fields.py

```diff
@@ -120,14 +121,22 @@ def schema(self):
 
     def get_related_url(self, obj):
         if self.related_url:
-            kwargs = resolve_params(obj, self.related_url_kwargs)
-            return self.related_url.format(**kwargs)
+            kwargs = resolve_params(obj, self.related_url_kwargs, default=self.default)
+            for key, value in copy(kwargs).items():
```

Comment:
Nitpick: Rather than copy the dictionary, this can be written more expressively as a dictionary comprehension:

```python
            params = resolve_params(obj, self.related_url_kwargs, default=self.default)
            non_null_params = {
                key: value for key, value in params.items()
                if value is not None
            }
            if non_null_params:
                return self.related_url.format(**non_null_params)
```